src/tree/registries/dockerHub/DockerHubNamespaceTreeItem.ts

```diff
@@ -57,7 +59,7 @@ export class DockerHubNamespaceTreeItem extends RegistryTreeItemBase {
 
     public async getDockerCliCredentials(): Promise<IDockerCliCredentials> {
         return {
-            registryPath: '',
```

Comment:
Can you verify that the Docker Login / Logout palette commands still work against Docker Hub? In the past we've never put the registry URL into the command line (the CLI assumes Docker Hub if no registry is given), but with this change, we will be.

Reply:
There is one thing I noticed though, monitoring the config file, if we don't add `index.docker.io`, we get the first line being added: 

![image](https://user-images.githubusercontent.com/40250218/211892763-4ba220de-8d81-499f-b9db-3a1fa8622868.png)

Would it be worth adding that whole url as the `registryPath` so they are consistent?

My url filter in ACA should be able to strip that back down to `index.docker.io` for the SDK so we'd still be good there as well.

Reply:
I'm not sure if the CLI treats them differently. The best thing to do would be to put a private image in Docker Hub, sign in using the new URL, and see if it can be pulled normally.

Reply:
Okay tested it, it looks like it is very specific about the url it wants.  I've updated the registry url accordingly!